src/datumaro/components/importer.py

```diff
@@ -165,3 +170,107 @@ def get_extractor_merger(self) -> Optional[Type[ExtractorMerger]]:
             Otherwise, use the return type to merge the extractors.
         """
         return None
+
+
+def with_subset_dirs(input_cls: Type[Importer]):
```

Comment:
Probably, these fixtures and related tests should be included in the PR.
https://github.com/openvinotoolkit/datumaro/blob/981717b1bcc4fe7bd848c04e9b8f6eb6ce5c266b/tests/unit/data_formats/conftest.py#L38